src/components/Pricing/PricingCenterBar.js

```diff
@@ -35,14 +35,14 @@ const PricingCenterBar = () => {
             </tr>
             <tr>
               <td class="table-body-text price-check  price-check-1 py-4">
-                Sellbrite
+                Inventory Flo
```

Comment:
Change **Inventory Flo** to **INVENTORY FLO**. Let it appear the same way it is on the logo.
Do the same to others.

Reply:
ok